src/syscheckd/src/db/src/file.cpp

```diff
@@ -495,6 +495,10 @@ FIMDBErrorCode fim_db_get_path(const char* file_path, callback_context_t callbac
             });
             retVal = FIMDB_OK;
         }
+        catch (const no_entry_found& err)
+        {
+            FIMDB::instance().logFunction(LOG_DEBUG_VERBOSE, err.what());
```

Comment:
why do you use LOG_DEBUG_VERBOSE instead LOG_ERROR?

Reply:
The error `There are more or 0 rows` was printed when the `GetFile` was successfully executed, but there wasn't any entry in the DB with that PK. This is shouldn't print an error, as there are some parts of FIM that will check if there is an entry, and handle properly the case that there is not an entry in the DB. 
Nevertheless, we are going to leave a verbose log to help the debugging process.